src/shared/ButtonProfile.jsx

```diff
@@ -0,0 +1,11 @@
+/*eslint-disable */
```

Comment:
You need to follow the eslint roles, see what it needs or what are the warnings and apply or fix,
instead of just ignoring.